features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -1287,12 +1287,22 @@ CREATE TABLE IF NOT EXISTS IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT(
     FOREIGN KEY (APPLICATION_ID) REFERENCES SP_APP(UUID) ON DELETE CASCADE
 );
 
-CREATE TABLE IF NOT EXISTS IDN_ORG_USER_ASSOCIATION(
-	SHARED_USER_ID VARCHAR(255) NOT NULL,
-	SUB_ORG_ID VARCHAR(36) NOT NULL,
-	REAL_USER_ID VARCHAR(255) NOT NULL,
-	USER_RESIDENT_ORG_ID VARCHAR(36) NOT NULL,
-	PRIMARY KEY (SHARED_USER_ID, SUB_ORG_ID, REAL_USER_ID, USER_RESIDENT_ORG_ID)
+CREATE TABLE IF NOT EXISTS TAG (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
```

Comment:
Don't see any value in this column.

Reply:
Removed the ID column since it has no use.